proto/api.proto

```diff
@@ -24,10 +24,6 @@ import "cockroach/proto/data.proto";
 import "cockroach/proto/errors.proto";
 import "gogoproto/gogo.proto";
 
-option (gogoproto.sizer_all) = true;
-option (gogoproto.marshaler_all) = true;
-option (gogoproto.unmarshaler_all) = true;
```

Comment:
Yeah, it's a halfway solution, but I think we can remove more options over time. Under what circumstances does a developer need to know we're using a custom protoc plugin?